employee-rostering-distribution/pom.xml

```diff
@@ -79,6 +79,24 @@
             <goals>
               <goal>repackage</goal>
             </goals>
+            <configuration>
+              <!--
+                Exclude backend and frontend artifacts. Since they are this module's dependencies,
+                the Spring Boot plugin would place them to BOOT-INF/lib but that's not what we want.
+                We only want to repackage the combination of these artifacts that have been unpacked
+                by dependency plugin.
+              -->
+              <excludes>
+                <exclude>
+                  <groupId>org.optaweb.employeerostering</groupId>
+                  <artifactId>employee-rostering-backend</artifactId>
+                </exclude>
+                <exclude>
+                  <groupId>org.optaweb.employeerostering</groupId>
+                  <artifactId>employee-rostering-frontend</artifactId>
+                </exclude>
+              </excludes>
```

Comment:
I think it might be missing the `exec` jar of backend:
```
    <exclude>
      <groupId>org.optaweb.employeerostering</groupId>
      <artifactId>employee-rostering-backend</artifactId>
      <classifier>exec</classifier>
    </exclude>
```

Reply:
It's a bit surprising but the exec jar is not included (even if I don't exclude it). I've checked again the content of the resulting single jar and it's not there. On this occasion I contained docs zip, so I excluded that as well.